src/vsr/replica.zig

```diff
@@ -6903,7 +6904,20 @@ pub fn ReplicaType(
                 return;
             }
 
-            // We may even be several views ahead and may now have a completely different pipeline.
+            if (self.commit_min != self.commit_max or self.commit_stage != .idle) {
```

Comment:
commit_min != commit_max is the actually fuzz failure here, but I think we need commmit_stage as well for the same reason we check it in repair